api/app/src/main/kotlin/packit/controllers/RoleController.kt

```diff
@@ -16,15 +16,15 @@ import java.net.URI
 
 @Controller
 @PreAuthorize("hasAuthority('user.manage')")
-@RequestMapping("/role")
+@RequestMapping("/roles")
 class RoleController(private val roleService: RoleService, private val userRoleService: UserRoleService)
 {
     @PostMapping()
     fun createRole(@RequestBody @Validated createRole: CreateRole): ResponseEntity<RoleDto>
     {
         val role = roleService.createRole(createRole)
 
-        return ResponseEntity.created(URI.create("/role/${role.id}")).body(role.toDto())
+        return ResponseEntity.created(URI.create("/roles/${role.id}")).body(role.toDto())
```

Comment:
Shouldn't the uri here use the role name rather than id as per GetMapping below? (If so, this was a bug before this changeset!)

```suggestion
        return ResponseEntity.created(URI.create("/roles/${role.name}")).body(role.toDto())
```

Reply:
Good catch yes. Thanks